src/main/server/start.sh

```diff
@@ -71,6 +77,11 @@ if [ "$OS_NAME" = "Darwin" ]; then
   SED_COMPATIBILITY='.bak'
 fi
 
+# Set JVM -Xms parameter
+if [ -n "$JVM_MEMORY_OPTIONS" ]; then
```

Comment:
This "if" statement is always true. No need to have if statement here. Just set it directly

```
JVM_MEMORY_OPTIONS="-Xms$DEFAULT_MIN_HEAP_SIZE"
```

Reply:
done